.circleci/config.yml

```diff
@@ -1,13 +1,83 @@
 version: 2
 jobs:
-  build:
-    docker:
-      - image: circleci/ruby:2.6.1-node-browsers
+  build_test:
+    resource_class: large
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      CHANGE_MINIKUBE_NONE_USER: true
+      GOPATH: /home/circleci/go
+      KUBECONFIG: /home/circleci/.kube/config
+      KUBERNETES_VERSION: v1.15.2
+      MINIKUBE_HOME: /home/circleci
+      MINIKUBE_VERSION: v1.2.0
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      SKAFFOLD_VERSION: v0.33.0
+      GO111MODULE: "on"
+      CI_SPLUNK_HEC_TOKEN: a6b5e77f-d5f6-415a-bd43-930cecb12959
+      CI_SPLUNK_HEC_TOKEN_PERF: a6b5e77f-d5f6-415a-bd43-930cecb12959
+      CI_SPLUNK_PASSWORD: helloworld
+      CI_HEC_PROTOCOL: http
     working_directory: ~/repo
     steps:
       - checkout
-      - setup_remote_docker:
-          reusable: true
+      - run:
+          name: Install Splunk
+          command: |
+            cd /tmp && wget -O splunk-7.3.2-c60db69f8e32-linux-2.6-amd64.deb 'https://www.splunk.com/bin/splunk/DownloadActivityServlet?architecture=x86_64&platform=linux&version=7.3.2&product=splunk&filename=splunk-7.3.2-c60db69f8e32-linux-2.6-amd64.deb&wget=true'
```

Comment:
set splunk-7.3.2-c60db69f8e32-linux-2.6-amd64 as an environment variable similar to MINIKUBE_VERSION